crypto/fipsmodule/sha/asm/sha512-x86_64.pl

```diff
@@ -555,12 +557,8 @@ ()
 .type	sha256_block_data_order_shaext,\@function,3
 .align	64
 sha256_block_data_order_shaext:
-.Lshaext_shortcut:
 .cfi_startproc
-#ifdef BORINGSSL_DISPATCH_TEST
```

Comment:
the dispatch test stuff should be left in, otherwise the test will fail on CPUs that support shaext (EC2 AMD instances for example)

Reply:
I thought about this and planned to revisit https://github.com/awslabs/aws-lc/pull/81

Will discuss with you offline.